metabase/metabase
# Extend color palette

Epic https://github.com/metabase/metabase/issues/22816

Add new colors to the color palette:
- 2 UI colors are separated from accent colors
- 16 new accent colors based on previous ones by default
- Adds a way to set them in Admin
- Adds new colors to color selectors
- Please note that `harmony` colors (used for automatic series color generation) are not changed yet

How to test:
- Go to Admin > Settings > Appearance
- Make sure it is possible to set main accent colors and accent color shades
- Go to a Table, Line, Map, Gauge questions and make sure it is possible to use new colors

<img width="613" alt="Screenshot 2022-05-30 at 19 27 15" src="https://user-images.githubusercontent.com/8542534/171031824-945e9ffb-c364-4de2-b9d6-3c99acddbbc9.png">
<img width="355" alt="Screenshot 2022-05-30 at 20 45 07" src="https://user-images.githubusercontent.com/8542534/171040455-4a647e87-33ac-4542-b487-78c5f693c954.png">
<img width="358" alt="Screenshot 2022-05-30 at 20 45 30" src="https://user-images.githubusercontent.com/8542534/171040461-e6f9e4a1-e3ab-4c80-ac9b-7a4c39878c4d.png">
<img width="356" alt="Screenshot 2022-05-30 at 20 45 48" src="https://user-images.githubusercontent.com/8542534/171040472-5817861a-b868-4d72-988f-132786a24bea.png">
<img width="354" alt="Screenshot 2022-05-30 at 20 46 17" src="https://user-images.githubusercontent.com/8542534/171040475-d79c62f6-b375-4a51-b0dc-79071718dc60.png">
<img width="359" alt="Screenshot 2022-05-30 at 20 46 34" src="https://user-images.githubusercontent.com/8542534/171040478-2ea34a10-7c3f-482d-b31c-e12ba699435b.png">
